apps/pixeloven/site/docs/getting-started/generator.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: generator
+title: Pixeloven Generators
+---
+
+Creating new apps, packages, or components within your project doesn't have to be a chore, we've provided generators to lend a hand.
+
+> Note: This section is a work in progress and should contain basic information for creating and working with generators
+
+## Overview
+
+To start any of the generators run the `generate` command
+
+```
+yarn pixeloven generate
+```
+
+The generators are somewhat opinionated and make a few assumptions about the organization of your project. See the instructions for each individual generator below for specifics.
+
+## Components
+
+1. To create a new component, run the `generate` command from the root of the host app or package and choose `Component` from the selection menu.
```

Comment:
Nit: on the numbers isn't there another syntax for auto numeration?

Reply:
I don't know of any other than using `1.` for every item in markdown. Maybe you're thinking of Bitbucket's own thing? https://www.markdownguide.org/basic-syntax#lists